flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBOptions.java

```diff
@@ -59,10 +59,15 @@
 		.defaultValue(1)
 		.withDescription("The number of threads (per stateful operator) used to transfer (download and upload) files in RocksDBStateBackend.");
 
-	/** This determines if compaction filter to cleanup state with TTL is enabled. */
+	/**
+	 * This determines if compaction filter to cleanup state with TTL is enabled.
+	 *
+	 * @deprecated the option will be removed in future and have no effect.
```

Comment:
```suggestion
	 * @deprecated the option will be removed in the future and should only be used if experiencing serious performance degradations.
```

Reply:
`should only be used >when< experiencing serious ...`

Reply:
Good point. Will change it while merging the PR.